platform/darwin/src/NSComparisonPredicate+MGLAdditions.mm

```diff
@@ -318,14 +318,32 @@ - (id)mgl_jsonExpressionObject {
         case NSNotEqualToPredicateOperatorType:
             op = @"!=";
             break;
+        case NSBetweenPredicateOperatorType: {
+            op = @"all";
+            NSArray *arguments = self.rightExpression.constantValue;
```

Comment:
`BETWEEN`, `CONTAINS`, and `IN` are most commonly written with aggregate expressions inline, not with arrays as constant values. For example, in the format string `a BETWEEN {1, 3}`, `{1, 3}` is an aggregate expression whose _collection_ is an array. It’s important to support aggregate expressions with these three operators, although supporting constant value arrays is also nice to have.